src/services/codefixes/addMissingAwait.ts

```diff
@@ -257,6 +257,7 @@ namespace ts.codefix {
                 sourceFile,
                 insertionSite.parent.expression,
                 createParen(createAwait(insertionSite.parent.expression)));
+            insertLeadingSemicolonIfNeeded(changeTracker, insertionSite.parent.expression, sourceFile);

```

Comment:
```suggestion
                insertLeadingSemicolonIfNeeded(changeTracker, insertionSite.parent.expression, sourceFile);
```

Reply:
The current indentation is correct (the preceding three lines are arguments to `replaceNode`; this is not